swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+   def __init__(self, id = None, condition = 0, age = None):
+      self.id = uuid.uuid1().int if id is None else id
+      self.category = "Item"
```

Comment:
```suggestion
```

The requirements for this class can be achieved without creating a new attribute for this class. Read my comment below for `get_category`